webcompat/templates/issue/issue-aside.html

```diff
@@ -1,13 +1,11 @@
-<aside class="wc-Issue-aside">
-  <div class="wc-Issue-information wc-Issue-information--fullWidth" id="js-Issue-aside">
-    {% include "issue/aside.jst" %}
-  </div>
-  <div class="wc-Issue-labels js-Issue-labels">
-    {% include "issue/issue-labels.jst" %}
-    {% include "web_modules/label-editor.jst" %}
-  </div>
-  <div class="wc-Issue-milestones js-Issue-milestones">
-    {% include "issue/issue-milestones.jst" %}
-    {% include "web_modules/milestone-editor.jst" %}
-  </div>
-</aside>
+<div id="js-Issue-aside">
+	{% include "issue/aside.jst" %}
```

Comment:
nit: whitespace. Do you have editorconfig installed?

https://github.com/webcompat/webcompat.com/blob/master/.editorconfig

(but, looks like we haven't defined indentation for HTML... I'll file a follow up issue for that).